elastic/kibana
# [Fleet] Restrict output type for Fleet Server

## Summary

Resolve #148896

Logstash is not a valid output type for Fleet Server.
That PR fix that by disalowing settings logstash as an output type for Fleet Server (in addition of the existing restriction for APM)


## UI Changes

<img width="946" alt="Screen Shot 2023-01-30 at 1 07 20 PM" src="https://user-images.githubusercontent.com/1336873/215562179-6b37226c-9416-4c81-a98e-0d1cd7399056.png">


## How to test?

1. Create a dummy logstash output no need to add real certificate
2. create a fleet server policy, you should not be able to set logstash as an output
